README.md

```diff
@@ -81,15 +103,41 @@ def create_changeset(%User{} = user, attrs) do
 end
 ```
 
-### 3: Enjoy!
+### 4: Configuration
+
+Add in your configuration file.
+
+```elixir
+config :task_after, global_name: TaskAfter
+```
+
+### 4.1: S3 Bucket Configuration(required for S3 storage only)
+
+Add in your configuration file.
```

Comment:
Maybe we can link ex_aws_s3 documentation for further info? https://github.com/ex-aws/ex_aws_s3

Reply:
Good idea